### PR TITLE
Feature/pelagos 4163 add fc aggregation in the search app

### PR DESCRIPTION
**Pull request by praneethpr**
Merged feature/PELAGOS-4164 into this, because both were using the same file and line to add global window variables

### CODE REVIEW

File: assets/js/vue/components/search/ResultSet.vue [hunk 1]
```diff
@@ -32,7 +32,8 @@
                     <aside class="col-lg-3">
                         <div class="card card-filter">
                             <Facet :facet-info="results.facetInfo.statusInfo" :facet-name="datasetStatus" v-on="$listeners" :formValues="formValues"/>
-                            <Facet :facet-info="results.facetInfo.fundingOrgInfo" :facet-name="fundingOrg" v-on="$listeners" :formValues="formValues"/>
+                            <Facet :facet-info="results.facetInfo.fundingCycleInfo" :facet-name="fundingCycle" v-on="$listeners" :formValues="formValues" v-if="showFundingCycleFacet()"/>
```
Comment (praneethpr) [inline assets/js/vue/components/search/ResultSet.vue:35]: Send facet title using facet-name prop

File: assets/js/vue/components/search/SearchForm.vue [hunk 2]
```diff
@@ -117,6 +117,15 @@
...
         },
+        computed: {
+            displayTextBlock: function () {
+                if (typeof window.PELAGOS_TEMPLATE_PROPS !== 'undefined') {
+                    return window.PELAGOS_TEMPLATE_PROPS.hideSearchTextBlock;
+                } else {
+                    return false;
+                }
+            }
+        },
```
Comment (mickel1138) [inline assets/js/vue/components/search/SearchForm.vue:128]: Isn't this from another PR, it should show up right?

Comment (praneethpr) [reply assets/js/vue/components/search/SearchForm.vue:128]: Yes, that's weird

Comment (praneethpr) [reply assets/js/vue/components/search/SearchForm.vue:128]: Did you commit anything after?

Comment (mickel1138) [reply assets/js/vue/components/search/SearchForm.vue:128]: This may be some Github weirdness, because this is on `develop`

**Pull request merged**